pkg/util/util_nix.go

```diff
@@ -1,5 +1,5 @@
-//go:build linux
-// +build linux
+//go:build !windows
```

Comment:
This is the first time I've seen the "nix" suffix being used. Is there a general convention that the Kubernetes project is using to support darwin?

Reply:
@thomastaylor312: can you explain where the `_nix.go` filename comes from? Is that being used elsewhere already?

Reply:
We can just call it `unix` if you prefer. I was just trying to indicate that this file wasn't for just linux, but any *nix system. I don't have any sort of a strong preference. Also no skin off of my teeth if you don't want to support darwin targets. This was just a proposal of something to do and can be shot down. As you can see, there was no change to code, so the restriction to linux build targets only was artificial and not for any specific code reason insofar as I can tell

Reply:
I (like @msau42 I guess) was just wondering. Personally I prefer "unix" over "nix". The latter might not be immediately recognizable as a placeholder for "unix or linux".

Kubernetes releases darwin/amd64 and darwin/arm64 client binaries, but not node binaries. I'm fine with merging this PR, but I would not actually build container images for Darwin.

Reply:
Yeah, no need to build a container image here. Let me rename the file real quick

Reply:
Ok, file has been renamed!